<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_task_type_pick</class>
 <widget class="QDialog" name="add_task_type_pick">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddTaskType</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type of Task:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>SuperBasicTask</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TrackedTask</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PrioritizedTask</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TimedTask</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>add_task_type_pick</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
